exposed all of the parameters in cli/autocontour

njneeteson opened the pull request:
I'm going to initiate this PR as a place to discuss but mark it as a draft until we're happy.

I've exposed all of the parameters in the CLI, but I still have to double check the defaults for XT2 and add defaults for uCT. Two problems:

1. I don't know what script is ran for the autocontour on XT2. When I go to modify evaluation script 28 in `uct_evaluation`, it doesn't provide any parameters in the interface at that level as far as I can see. I also can't find the script to look inside of it myself - it says that it will run `UE:IPLV6_AUTOK.COM`but I looked in the `COM` folder and it's not there so I'm not sure where that script is located or if that's an alias filename. There's a lot of autocontour scripts in there and I only want to look at the one that actually gets used for standard radius/tibia largest bone autocontouring.
2. I don't know how to log into the uCT system.

stevenkboyd commented:
There is a video tutorial on how the uct_evaluation works available on the BOYD drive. I did it a while ago, but it shows how a script is called, and how to set the fixed parameters. Maybe that'll help? As for the MICROCT system, you can log in at invivo@10.148.64.62 and use the standard pw: made...

njneeteson commented:
Removed the arg for switching to uct defaults. All of the parameters used in the pipeline have been exposed in the arg parser. As discussed elsewhere, this is based on Helen Buie's original algorithm for Xt1 and is not reflective of the current algorithms used in the semi-automated protocol for XT2. Implementing the XT2 algorithm might be done in a separate PR in the future (and added as a separate utility)
